apps/web/components/ui/PurchaseCard/PurchaseCard.vue

```diff
@@ -196,10 +196,11 @@ const handleAddToCart = async () => {
     return;
   }
 
+  const orderProperties = getPropertiesForCart();
   const params = {
     productId: Number(productGetters.getId(product.value)),
     quantity: Number(quantitySelectorValue.value),
-    basketItemOrderParams: getPropertiesForCart(),
+    basketItemOrderParams: orderProperties.length > 0 ? orderProperties : undefined,
```

Comment:
can't we rewrite the getOrderProperties method to return undefined if there are no properties? Or am i missing something?

Reply:
Then we need to update "getPropertiesPrice" too.

Reply:
already did in the changes from this morning. Just didn't pushed yet. But not sure, what to do with "getPropertiesPrice", when properties = undefined. Right now im just returning null.